Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_sdio_slot0.cpp

```diff
@@ -191,81 +179,52 @@ IPCCommandResult CWII_IPC_HLE_Device_sdio_slot0::IOCtl(u32 _CommandAddress)
     INFO_LOG(WII_IPC_SD, "IOCTL_GETSTATUS. Replying that SD card is %s%s",
              (m_Status & CARD_INSERTED) ? "inserted" : "not present",
              (m_Status & CARD_INITIALIZED) ? " and initialized" : "");
-    Memory::Write_U32(m_Status, BufferOut);
+    Memory::Write_U32(m_Status, request.buffer_out);
     break;
 
   case IOCTL_GETOCR:
     INFO_LOG(WII_IPC_SD, "IOCTL_GETOCR");
-    Memory::Write_U32(0x80ff8000, BufferOut);
+    Memory::Write_U32(0x80ff8000, request.buffer_out);
     break;
 
   default:
-    ERROR_LOG(WII_IPC_SD, "Unknown SD IOCtl command (0x%08x)", Cmd);
+    ERROR_LOG(WII_IPC_SD, "Unknown SD IOCtl command (0x%08x)", request.request);
     break;
   }
 
-  if (ReturnValue == RET_EVENT_REGISTER)
+  if (return_value == RET_EVENT_REGISTER)
   {
-    // async
-    m_event.addr = _CommandAddress;
-    Memory::Write_U32(0, _CommandAddress + 0x4);
     // Check if the condition is already true
     EventNotify();
     return GetNoReply();
```

Comment:
Yes, this is intended. EventNotify() writes the proper return code, so writing 0 (which is a dummy code here) is not needed.